### PR TITLE
{lib} [intel-2014b] lxml 3.4.2 (REVIEW) 

### PR DESCRIPTION
**Pull request by hajgato**
lxml 3.4.2 with intel2014b plus deps.

### CODE REVIEW

**Comment by boegel** [comment]
@hajgato: I'm OK with merging these (if they work ;)), but why `intel/2014b`?

File: easybuild/easyconfigs/l/libxslt/libxslt-1.1.28-intel-2014b.eb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+source_urls = [
+    'http://xmlsoft.org/sources/',
+    'http://xmlsoft.org/sources/old/'
+]
+
+sources = [SOURCELOWER_TAR_GZ]
+
+dependencies = [
+    ('zlib', '1.2.8'),
+    ('libxml2', '2.9.1'),
```
Comment (boegel) [inline easybuild/easyconfigs/l/libxslt/libxslt-1.1.28-intel-2014b.eb]: the more recent easyconfigs use `libxml2 v2.9.2`, so bump version?

File: easybuild/easyconfigs/l/lxml/lxml-3.4.2-intel-2014b-Python-2.7.8.eb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+python = 'Python'
+pythonversion = '2.7.8'
+pyshortver = '.'.join(pythonversion.split('.')[:2])
+
+versionsuffix = "-%s-%s" % (python, pythonversion)
+
+dependencies = [
+    (python, pythonversion),
+    ('libxml2', '2.9.1'),
```
Comment (boegel) [inline easybuild/easyconfigs/l/lxml/lxml-3.4.2-intel-2014b-Python-2.7.8.eb]: use `2.9.2`?

**Comment by hajgato** [comment]
@boegel: I got an user request. I know it is old, but I think it still worth to share.

**Comment by boegel** [comment]
@hajgato: you should try to push your users to use newer toolchains (unless they have a good reason not to), otherwise you'll be supported old cruft for years to come...

But sure, it's still useful to merge it in if it works, no harm done.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/633f7bb36f0ab51d76b1 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @hajgato!

**Pull request merged**